manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md

```diff
@@ -10,30 +10,69 @@ products:
 
 # Configure a lifecycle policy [set-up-lifecycle-policy]
 
-For {{ilm-init}} to manage an index, a valid policy must be specified in the `index.lifecycle.name` index setting.
+An [{{ilm}}](/manage-data/lifecycle/index-lifecycle-management.md) ({{ilm-init}}) policy defines how your indices are managed over time, automating when and how they transition as they reach a certain age, size, or number of documents.
 
-To configure a lifecycle policy for [rolling indices](rollover.md), you create the policy and add it to the [index template](../../data-store/templates.md).
+There a few things to note about how an {{ilm-init}} policy works:
 
-To use a policy to manage an index that doesn’t roll over, you can specify a lifecycle policy when you create the index, or apply a policy directly to an existing index.
+* For {{ilm-init}} to manage an index, you need to specify a valid policy in the `index.lifecycle.name` index setting.
 
-{{ilm-init}} policies are stored in the global cluster state and can be included in snapshots by setting `include_global_state` to `true` when you [take the snapshot](../../../deploy-manage/tools/snapshot-and-restore/create-snapshots.md). When the snapshot is restored, all of the policies in the global state are restored and any local policies with the same names are overwritten.
+* To configure a lifecycle policy for [rolling indices](rollover.md) or data streams, you create the policy and add it to the [index template](../../data-store/templates.md).
```

Comment:
...+ See [Automate Rollover](https://www.elastic.co/docs/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover).

Reply:
I added:
`Refer to the [Tutorial: Automate rollover](/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover.md) for a detailed walkthrough.`